context/app/static/js/components/detailPage/Attribution/Attribution.tsx

```diff
@@ -1,32 +1,95 @@
 import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
-
-import { FlexPaper } from './style';
-import SectionItem from '../SectionItem';
+import SummaryPaper from 'js/shared-styles/sections/SectionPaper';
+import LabelledSectionText from 'js/shared-styles/sections/LabelledSectionText';
+import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+import InfoAlert from 'js/shared-styles/alerts/InfoAlert';
 
 function Attribution() {
   const {
-    entity: { group_name, created_by_user_displayname, created_by_user_email, entity_type },
+    entity: {
+      group_name,
+      created_by_user_displayname,
+      created_by_user_email,
+      entity_type,
+      processing,
+      creation_action,
+      descendants,
+    },
   } = useFlaskDataContext();
 
-  return (
-    <DetailPageSection id="attribution">
-      <SectionHeader iconTooltipText={`Information about the group registering this ${entity_type?.toLowerCase()}.`}>
-        Attribution
-      </SectionHeader>
-      <FlexPaper>
-        <SectionItem label="Group">{group_name}</SectionItem>
-        <SectionItem label="Registered by" ml>
+  const isProcessedDataset = entity_type === 'Dataset' && processing === 'processed';
+  const isVisLiftedDataset = descendants?.find((descendant) => descendant.dataset_type === 'Histology [Image Pyramid]');
+  const isHiveProcessedDataset = isProcessedDataset && creation_action === 'Central Process';
+  const isSupportDataset = entity_type === 'Support';
+
+  const showContactAndAlert = isHiveProcessedDataset ?? isSupportDataset;
+  const showRegisteredBy = !(isProcessedDataset ?? isVisLiftedDataset ?? isHiveProcessedDataset ?? isSupportDataset);
+
+  const tooltips = {
+    group: 'This is the group that provided the raw dataset.',
+    contact: 'This is the contact for this data.',
+  };
+
+  const hiveInfoAlertText = `The data provided by the ${group_name} Group was centrally processed by HuBMAP. The results of this processing are independent of analyses conducted by the data providers or third parties.`;
+
+  const sections: {
+    label: string;
+    text: React.ReactNode;
+    tooltip?: string;
+  }[] = [
+    {
+      label: 'Group',
+      text: group_name,
+      tooltip: showRegisteredBy ? undefined : tooltips.group,
+    },
+  ];
+
+  if (showContactAndAlert) {
+    sections.push({
+      label: 'Contact',
+      text: <ContactUsLink> HuBMAP Help Desk </ContactUsLink>,
+      tooltip: tooltips.contact,
+    });
+  } else if (showRegisteredBy) {
+    sections.push({
+      label: 'Registered by',
+      text: (
+        <Stack spacing={1}>
           {created_by_user_displayname}
           <EmailIconLink email={encodeURI(created_by_user_email)} iconFontSize="1.1rem">
             {created_by_user_email}
           </EmailIconLink>
-        </SectionItem>
-      </FlexPaper>
+        </Stack>
+      ),
+    });
+  }
```

Comment:
I would consider pulling this logic out into a hook or helper component, which would make it a bit easier to migrate this logic over to the unified views. I also prefer to put iterative logic into `useMemo`s when possible as a minor performance gain.

Something like this:

```tsx
function useAttributionSections(
	group_name: string, 
	showRegisteredBy: boolean, 
	created_by_user_displayname: string, 
	created_by_user_email: string) {
		return  useMemo(() => {
			// section generation logic in here
		}, [group_name, showRegisteredBy, created_by_user_displayname, created_by_user_email]}
}
```